week1/homework/src/server.js

```diff
@@ -1,20 +1,48 @@
+
 'use strict';
 
 const http = require('http');
 
-/* `createServer` MUST return an instance of `http.Server` otherwise the tests
- * will fail.
- */
+function printData(state, response) {
```

Comment:
you dont have to change this, but good thing  to keep in mind is that naming is important, and giving accurate and descriptive names  for functions and variables is good practice :)